Database/Migrations/2021_07_31_161559_skipper_migrations_2021073116155945.php

```diff
@@ -0,0 +1,1440 @@
+        });
+        Schema::table('vh_theme_locations', function (Blueprint $table) {
+            $table->bigInteger('vh_theme_id')->nullable(true)->unsigned()->after('id')->change();
+        });
+        Schema::table('vh_theme_template_fields', function (Blueprint $table) {
+            $table->bigInteger('id')->autoIncrement()->unsigned()->change();
+        });
+        Schema::table('vh_theme_template_fields', function (Blueprint $table) {
+            $table->bigInteger('vh_template_id')->nullable(true)->unsigned()->after('id')->change();
+        });
+        Schema::table('vh_theme_template_fields', function (Blueprint $table) {
+            $table->bigInteger('vh_theme_id')->nullable(true)->unsigned()->after('vh_template_id')->change();
+        });
+        Schema::table('vh_theme_templates', function (Blueprint $table) {
+            $table->bigInteger('id')->autoIncrement()->unsigned()->change();
+        });
+        Schema::table('vh_theme_templates', function (Blueprint $table) {
+            $table->bigInteger('vh_theme_id')->nullable(true)->unsigned()->after('id')->change();
+        });
+        Schema::table('vh_themes', function (Blueprint $table) {
+            $table->bigInteger('id')->autoIncrement()->unsigned()->change();
+        });
+        Schema::table('vh_user_authorizations', function (Blueprint $table) {
+            $table->bigInteger('id')->autoIncrement()->unsigned()->change();
+        });
+        Schema::table('vh_user_authorizations', function (Blueprint $table) {
+            $table->bigInteger('vh_user_id')->nullable(true)->unsigned()->after('id')->change();
+        });
+        Schema::table('vh_user_roles', function (Blueprint $table) {
+            $table->bigInteger('id')->autoIncrement()->unsigned()->change();
+        });
+        Schema::table('vh_user_roles', function (Blueprint $table) {
+            $table->bigInteger('vh_role_id')->nullable(true)->unsigned()->after('vh_user_id')->change();
+        });
+        Schema::table('vh_user_roles', function (Blueprint $table) {
+            $table->bigInteger('vh_user_id')->nullable(true)->unsigned()->after('id')->change();
+        });
+        Schema::table('vh_users', function (Blueprint $table) {
+            $table->bigInteger('id')->autoIncrement()->unsigned()->change();
+        });
+        /* Reordering of existing columns is not supported by Laravel Eloquent.
+        Schema::table('vh_theme_blocks', function (Blueprint $table) {
+            $table->moveColumnAfter('vh_theme_id', 'id');
+        });
+        */
+        
+        /* But you can create your own migrations with following SQL query based on your target SQL engine.
+        DB::transaction(function () {
+            DB::statement('ALTER TABLE `vh_theme_blocks` MODIFY `vh_theme_id` [COLUMN_DEF] AFTER `id`');
+        });
+        */
+        /* Reordering of existing columns is not supported by Laravel Eloquent.
```

Comment:
@TomHAnderson do I need to review your comments?

Reply:
No.  I'll explain this as part of removing the .skipper file